crates/aptos-faucet/ts-client/src/models/RejectionReasonCode.ts

```diff
@@ -0,0 +1,15 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+export enum RejectionReasonCode {
+    HEHE = 'Hehe',
```

Comment:
:no_mouth:

Reply:
lol

Reply:
lol yeah there is actually a good reason for this, I'll tell you later